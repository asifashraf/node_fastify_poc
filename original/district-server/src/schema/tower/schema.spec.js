const {
  fetchGraphQL,
  // openInGraphiql,
} = require('../../lib/test-util');

test('can query for towers', async () => {
  const query = `{
    config {
      towers {
        id
        name
      }
    }
  }
  `;

  // openInGraphiql(query);
  const result = await fetchGraphQL(query);
  expect(result).toMatchSnapshot();
});
