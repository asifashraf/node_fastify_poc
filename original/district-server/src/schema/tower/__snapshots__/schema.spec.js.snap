// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can query for towers 1`] = `
Object {
  "data": Object {
    "config": Object {
      "towers": Array [
        Object {
          "id": "c7caa967-dea5-4033-ba9d-6ccdfef3ed3e",
          "name": "25 February Tower",
        },
        Object {
          "id": "caff2b2f-70c7-4bf0-9c77-55a58c3b7fb3",
          "name": "26 February Tower",
        },
        Object {
          "id": "0d21e02c-9cf5-46f6-9667-8531c621be03",
          "name": "Ahli Bank of Kuwait",
        },
        Object {
          "id": "bf5b753f-2e3c-429f-b8c3-59b567ee46ef",
          "name": "Ahli United Bank",
        },
        Object {
          "id": "0d2f72c7-915a-4063-ba06-da1b816d3b8a",
          "name": "Al Amiri Hospital",
        },
        Object {
          "id": "d7316430-def1-46a1-9844-a88b146237e6",
          "name": "Al Hamra Tower",
        },
        Object {
          "id": "dbf87cf2-5795-465e-b59c-5528220b1e87",
          "name": "Al Rayah Center",
        },
        Object {
          "id": "62536f50-cb56-4493-a364-9e1f2507cf5f",
          "name": "Al Rayah Tower",
        },
        Object {
          "id": "c0c7f9ab-612d-4fc0-86bd-818a838d3c47",
          "name": "Al Sahab Tower",
        },
        Object {
          "id": "ca53cee9-5943-494d-a466-b602ef79a8da",
          "name": "Al Salhiya Tower",
        },
        Object {
          "id": "307e8619-ff3d-462b-a671-26d5851555cc",
          "name": "Baitak Tower",
        },
        Object {
          "id": "60a5e18f-3669-457a-adde-4cccce01e655",
          "name": "Boubyan Bank",
        },
        Object {
          "id": "b9ca6e73-0e56-4d28-b682-ffbd2225b7b8",
          "name": "Burgan Bank",
        },
        Object {
          "id": "5ff43ba8-9357-4fa9-9a5f-489ab644ba72",
          "name": "Central Bank of Kuwait",
        },
        Object {
          "id": "b46ff829-814d-48c2-b981-a6d675d2432b",
          "name": "Commercial Bank of Kuwait",
        },
        Object {
          "id": "871e1f9c-62e2-4167-b1b2-622cd37f550e",
          "name": "Crystal Tower",
        },
        Object {
          "id": "bcb20f18-b426-46d5-8349-c167b001d919",
          "name": "Gulf Bank",
        },
        Object {
          "id": "3fe5597e-0c40-4e9f-aada-dc5cc4b72193",
          "name": "Gulf Investment Corporation",
        },
        Object {
          "id": "5f8199fa-9fcb-416b-9f2c-abade719a1e9",
          "name": "Industrial Bank of Kuwait",
        },
        Object {
          "id": "ef264401-9756-431c-9e29-6c9b120a4a99",
          "name": "KBT",
        },
        Object {
          "id": "af2963b9-6498-46e9-bc78-eb5ad882cb7f",
          "name": "KFH Head Office",
        },
        Object {
          "id": "78c5f9e7-57bb-4aaf-90d3-f3b18bdd722d",
          "name": "Kipco Tower",
        },
        Object {
          "id": "e3cc9fa1-6a99-4fbd-aa75-d3fcc5a93436",
          "name": "Kuwait International Bank",
        },
        Object {
          "id": "aef537d4-8089-48ad-874e-6dfc1275caaf",
          "name": "Law Courts Palace",
        },
        Object {
          "id": "b4100d24-23c7-420f-bf99-08fbf131fd4a",
          "name": "Mazaya 1 Tower",
        },
        Object {
          "id": "8cf99d47-a784-4021-92a0-5e6ff1c532a4",
          "name": "Mazaya 2 Tower",
        },
        Object {
          "id": "48a906ce-777f-47aa-b7da-26690405d1c2",
          "name": "Mazaya 3 Tower",
        },
        Object {
          "id": "018e28f2-63b6-4f2b-bc48-ce33f97aeddb",
          "name": "Ministries Complex",
        },
        Object {
          "id": "682a4481-b88d-4147-bcde-631f718949e9",
          "name": "National Bank of Kuwait",
        },
        Object {
          "id": "5606122f-749a-4554-a776-ee795159beaf",
          "name": "Ooredoo Tower",
        },
        Object {
          "id": "7d3692e1-6970-457a-a313-643f3e3058a5",
          "name": "Sanabil Tower",
        },
        Object {
          "id": "a38c42cd-ce8e-41de-bd03-abfbbc9faefa",
          "name": "Seif Palace",
        },
        Object {
          "id": "fcd11fa0-18e2-457e-a3a7-638c9a75eb18",
          "name": "Tijariya Tower",
        },
      ],
    },
  },
}
`;
